### ISSUE TITLE
Already set valid api key is overwritten with default api key.

### ISSUE THREAD

**Issue by bitbeans** [exceptionless/Exceptionless.Net#19]
I was setting my api key in code and I noticed that it was being overwritten on the call to register:

```csharp
ExceptionlessClient.Default.Configuration.ApiKey = "My Key";
ExceptionlessClient.Default.Register();
// Api Key is now set to API_KEY_HERE
```

My app config had:

```xml
<exceptionless apiKey="API_KEY_HERE" />
```

**Comment by niemyjski** [comment]
This fix will be deployed after #22 has been fixed.

**Comment by niemyjski** [comment]
We are in the process of pushing [v3.1.0](https://github.com/exceptionless/Exceptionless.Net/releases/tag/v3.1.0) out to NuGet!
